api/models/review.js

```diff
@@ -0,0 +1,31 @@
+const mongoose = require('mongoose');
+
+const reviewSchema = new mongoose.Schema({
```

Comment:
lets add an ID to this model since we could have ratings with the same name or author :)